README.md

```diff
@@ -300,33 +300,40 @@ A TestCase Context has to implement this interface
 ```go
 
 type TestCaseContext interface {
-	BuildContext(tc *TestCase) (map[string]interface{}, error)
+	Init() error
+	Close() error
+	SetTestCase(tc TestCase)
+	GetName() string
 }
 ```
 
 Example
 
 ```go
 // Context Type name
-const Name = "foo"
-const ContextFooKey = "foo"
-const ContextBarKey = "bar"
+const Name = "default"
 
 // New returns a new TestCaseContext
 func New() venom.TestCaseContext {
-	return &TestCaseContext{}
+	ctx := &TestCaseContext{}
+	ctx.Name = Name
+	return ctx
 }
 
-// TestCaseContex represents the context of a testcase
+// TestCaseContext represents the context of a testcase
```

Comment:
Can you rename this struct ?  it's little confusing with TestCaseContext Interface